runtime/android/BUILD.gn

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/rules.gni")
+
+shared_library("libxwalkcore") {
+  sources = [
+    "//xwalk/runtime/app/android/xwalk_entry_point.cc",
```

Comment:
If all the files are in a different directory, the entire target should probably be in `xwalk/runtime/app/android/BUILD.gn` instead.